getting_started/scripting/c_sharp/c_sharp_style_guide.rst

```diff
@@ -31,12 +31,16 @@ Formatting conventions
 ----------------------
 
 * Use line feed (**LF**) characters to break lines, not CRLF or CR.
+* Use one line feed character at the end of each file, except for `csproj` files.
 * Use **UTF-8** encoding without a `byte order mark <https://en.wikipedia.org/wiki/Byte_order_mark>`_.
 * Use **4 spaces** instead of tabs for indentation (which is referred to as "soft tabs").
 * Consider breaking a line into several if it's longer than 100 characters.
 
+Code structure
```

Comment:
Same comment as with the other guide, you could use `Formatting conventions` here. And then it's good to merge as far as I'm concerned.

Reply:
Sorry, I don't understand what you want me to do. I already split code structure and formatting conventions, and now this code structure section should be formatting conventions? It might be easier for you to just merge and fix it since I don't understand how it should look like.

Reply:
No worries. It might be me. I mean that, to me, the entire document only covers formatting conventions at the moment.

My understanding of "code structure" is that it refers to what the program does, the structure of the class, rather than how the document looks. Things like the order in which you call functions in a given code block. Like code architecture, but at a smaller scale. That's why to me "Code structure" doesn't fit in these documents.